scalding-core/src/test/scala/com/twitter/scalding/source/typedtext/TypedTextTest.scala

```diff
@@ -0,0 +1,24 @@
+package com.twitter.scalding.source.typedtext
+
+import com.twitter.scalding._
+
+import org.scalatest.FunSuite
+import org.scalatest.junit.JUnitRunner
+
+case class Test1(a: Int, b: Long, c: Option[Double])
+case class Test2(one: Test1, d: String)
+
+class TypedTextTest extends FunSuite {
+  test("Test with a flat tuple") {
+    val source = TypedText.tsv[Test1]("myPath")
+    assert(source.sourceFields.size == 3)
+  }
+  test("Test with a nested tuple") {
+    val source = TypedText.tsv[Test2]("myPath")
```

Comment:
does this work with a tuple `(Int, Int)` or a raw type like `String`? We need that if we are going to replace `TypedTsv`.